tests/test_joint.py

```diff
@@ -207,12 +221,9 @@ def test_hingejoint():
         rod2.external_forces[..., rod2_index], -1 * contactforce, atol=Tolerance.atol()
     )
 
-    linkdirection = (
-        rod2.position_collection[..., rod2_index + 1]
-        - rod2.position_collection[..., rod2_index]
-    )
```

Comment:
@armantekinalp I changed the computation of the link direction so that it is compatible with rigid bodies. The computation using the position of multiple nodes only works for rods.